### PR TITLE
Sec autoinjector tweaks

### PR DESCRIPTION
**Pull request by a-person5660**
## About the PR
expanded on the security autoinjector, its no longer purely for epinephrine and now can take a few types of cartriges.
the cooldown for using them is now 20 seconds instead of 5 minutes.
also added ephedrine and ichor cartridges and the autoinjector itself to the autolathe when emagged at a cost of a bunch of gold, silver, plasma, etc.

the new cartridges are: bicaridine (7u bicar, 3u tranexamic acid), dermaline (10u dermaline), ephedrine (10u ephedrine, also syndie only), ichor ( 7u ichor, 3u tranexamic acid, also synyide only) also increased the base cost cause it was kinda low.

## Why / Balance
epi pens are more abundant and contain more reagents than the autoinjector so i think giving it some extra utility is nice.
giving it more emag interaction is always fun.

lmk if im stupid and this is actually unbalanced as fuck, i made this all at 4 in the morning.

**Changelog**

:cl:
- add: Added new cartridges for the Autoinjector: Bicaridine, Dermaline, Ephedrine, and Ichor (Ephedrine and Ichor are Syndicate only).
- tweak: You can now print Bicaridine and Dermaline cartridges in the Security Autolathe.
- tweak: Using an Emag on a Security Autolathe now unlocks hidden Autoinjector cartridge recipes: Ephedrine and Ichor.
- tweak: The cooldown for using the Autoinjector is now 40 seconds instead of 5 minutes.

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/tags.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   id: BolaEnergy
 
 - type: Tag
-  id: CartridgeEpinephrine
+  id: AutoinjectorCartridge
```
[the remark this answers is not in the document]
Comment (a-person5660) [reply Resources/Prototypes/_Goobstation/tags.yml:8]: the usage of CartridgeEpinephrine elsewhere is as an ID and not a tag so this is not an issue.

File: Resources/Prototypes/_Goobstation/Recipes/Lathes/security.yml [hunk 2]
```diff
@@ -19,3 +19,51 @@
...
+  id: SecHypo
+  result: SecHypo
+  completetime: 3
+  materials:
+    Steel: 1500
+    Plastic: 800
+    Glass: 1200
+    Plasma: 500
+    Gold: 500
+    Silver: 500
```
[the remark this answers is not in the document]
Comment (a-person5660) [reply Resources/Prototypes/_Goobstation/Recipes/Lathes/security.yml:69]: already done in the prototype descriptions

**Comment by JoeHammad1844** [comment]
would be cool if paramedics got this tbh

**Comment by Aviu00** [comment]
20 seconds is too low, people would inject epi constantly, make it higher.

File: Resources/Textures/_Goobstation/Objects/Specific/Medical/cartridge.rsi/meta.json [hunk 3]
```diff
@@ -8,7 +8,22 @@
   },
   "states": [
     {
-      "name": "icon"
+      "name": "cartridge"
```
Comment (Aviu00) [inline Resources/Textures/_Goobstation/Objects/Specific/Medical/cartridge.rsi/meta.json:11]: Change copyright if you added new sprites.

Comment (a-person5660) [reply Resources/Textures/_Goobstation/Objects/Specific/Medical/cartridge.rsi/meta.json:11]: i forgor

**Comment by a-person5660** [comment]
> 20 seconds is too low, people would inject epi constantly, make it higher.

how about i up the cost of the cartridges?

**Comment by Aviu00** [comment]
> > 20 seconds is too low, people would inject epi constantly, make it higher.
> 
> how about i up the cost of the cartridges?

When silo gets merged cost wouldn't really be a problem.

**Comment by Aviu00** [comment]
> good point, would 30 seconds be good?

Make it 40 seconds.

**Comment by a-person5660** [comment]
done

**Comment by Piras314** [comment]
you need to make better CLs and update them, I did it for you this time

**Pull request merged**